ui/lib/core/addon/components/replication-page.js

```diff
@@ -12,6 +12,7 @@ const MODE = {
 export default Component.extend({
```

Comment:
would you be willing to add JSDoc comments for this component? there are a lot of very similar properties in here and i think it'll help us later to remember what each thing is.

Reply:
Sure thing.  However, the replication-page only takes in the cluster model, so the write up is pretty small.  I'll do the same for the replication-dashboard, which takes in a couple more properties.